release.sh

```diff
@@ -201,9 +201,7 @@ if [ $dodist -eq 0 ]; then
     # global cache here to ensure we get the right thing.
     yarn cache clean
     yarn install
-    # We haven't tagged yet, so tell the dist script what version
-    # it's building
-    DIST_VERSION="$tag" yarn dist
+    yarn build
```

Comment:
hmm, won't this break the riot-web release since riot-web uses the same release script and uses the dist step to make the built tarball?

Reply:
What? Really? Why is riot using the js-sdk release script to release itself?

This was removed because it's a no-op in the js-sdk.

Reply:
yep, this is a shared release script between the three projects

Reply:
That's a bit awkward. I guess the fastest way forward is to fix it and add a warning to the top rather than refactor the thing out to the layers.